src/llscan.cc

```diff
@@ -474,6 +545,34 @@ void FindReferencesCmd::ReferenceScanner::PrintRefs(
 }
 
 
+void FindReferencesCmd::PropertyScanner::PrintRefs(
+    SBCommandReturnObject& result, v8::JSObject& js_obj, v8::Error& err) {
+  v8::Value::InspectOptions inspect_options;
+  inspect_options.detailed = false;
+
+  // (Note: We skip array elements as they don't have names.)
+
+  // Walk all the properties in this object.
+  // We only create strings for the field names that match the search
+  // value.
+  std::vector<std::pair<v8::Value, v8::Value>> entries = js_obj.Entries(err);
+  if (err.Success()) {
```

Comment:
That removes some indenting. (Not sure how it ended up like that.)
I've gone with `if (err.Fail())` to match the rest of the code in llscan.cc.